src/sql/SqlService.ts

```diff
@@ -378,11 +380,24 @@ export class SqlServiceImpl implements SqlService {
         return this.executeStatement(sqlStatement);
     }
 
+    /**
+     * Sends a close request on a connection for an SQL result using its query id.
+     * @param connection The connection the request will be sent to
+     * @param queryId The query id that defines the SQL result
+     * @internal
```

Comment:
I see that this class is internal, do we need to mark methods as internal again?